runtime/centrifuge/src/migrations.rs

```diff
@@ -63,37 +63,6 @@ pub type UpgradeCentrifuge1025 = (
 		PoolCurrencyAnemoy,
 		LocalCurrencyIdUsdc,
 	>,
-	// Register trading pairs one by one
-	runtime_common::migrations::local_currency::add_bidirectional_trading_pair::Migration<
-		super::Runtime,
-		UsdcDot,
-		LocalCurrencyIdUsdc,
-		MinOrderAmount,
-	>,
-	runtime_common::migrations::local_currency::add_bidirectional_trading_pair::Migration<
-		super::Runtime,
-		UsdcEth,
-		LocalCurrencyIdUsdc,
-		MinOrderAmount,
-	>,
-	runtime_common::migrations::local_currency::add_bidirectional_trading_pair::Migration<
-		super::Runtime,
-		UsdcBase,
-		LocalCurrencyIdUsdc,
-		MinOrderAmount,
-	>,
-	runtime_common::migrations::local_currency::add_bidirectional_trading_pair::Migration<
-		super::Runtime,
-		UsdcArb,
-		LocalCurrencyIdUsdc,
-		MinOrderAmount,
-	>,
-	runtime_common::migrations::local_currency::add_bidirectional_trading_pair::Migration<
-		super::Runtime,
-		UsdcCelo,
-		LocalCurrencyIdUsdc,
-		MinOrderAmount,
-	>,
```

Comment:
We still need a migration to kill the storage or trading pairs... WDYT @wischli ?

Reply:
Makes sense 👍🏻

Reply:
After another thought, I think it's safer to just reset completely `pallet-order-book`. We should close, of course, the current pending order before the RU.

Reply:
I agree!